packages/components/src/FormatFile/FormatFile.css

```diff
@@ -51,12 +51,61 @@
   margin: 0 auto;
 }
 
-.progress {
+.fileContentWrapper {
+  display: flex;
+  width: inherit;
+  height: inherit;
+  align-items: center;
```

Comment:
If you are using something like `margin: 0 auto;` on the icon then properties like align-items, justify-content won't do anything. I can't find where this is being set in the file, but it is there in Chrome inspector. The reason why the margin property isn't working to center is because this block has `flex-direction: column`, this means (vertical flexibility) so combined with `0 auto` won't center vertically because the `0` needs to be `auto` as well.

To my comment above, I believe the previous structure worked, but we are having to do this styling just for the thumbnail variant.